<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>837</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>700</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../_img/login.png</normaloff>../_img/login.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>35</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>550</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(16, 16, 16);
border-radius: 10px;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLineEdit" name="lineEdit">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>248</y>
            <width>280</width>
            <height>50</height>
           </rect>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI Black</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit{
	border: 2px solid rgb(85, 0, 255);
	border-radius: 5px;
	padding: 15px;
	background-color:  rgb(255, 255, 255);
	color: rgb(0, 0, 0);
}

QLineEdit:hover{
	border: 3px solid rgb(148, 94, 255);
}

QLineEdit:focus{
	border: 2px solid rgb(255, 20, 63);
}</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="placeholderText">
           <string>LOGIN</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_2">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>310</y>
            <width>280</width>
            <height>50</height>
           </rect>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI Black</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit{
	border: 2px solid rgb(85, 0, 255);
	border-radius: 5px;
	padding: 15px;
	background-color:  rgb(255, 255, 255);
	color: rgb(0, 0, 0);
}

QLineEdit:hover{
	border: 3px solid rgb(148, 94, 255);
}

QLineEdit:focus{
	border: 2px solid rgb(255, 20, 63);
}</string>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="placeholderText">
           <string>SENHA</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_2">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>429</y>
            <width>280</width>
            <height>40</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background-color: rgb(85, 0, 255);
	background-position: center;
	border-radius:5px;
}
QPushButton:hover{
	background-color: rgb(85, 0, 255);
	border: 3px solid rgb(148, 94, 255);
}
QPushButton:pressed{
	border: 2px solid rgb(255, 255, 255);
	color: rgb(0, 0, 0);
}</string>
          </property>
          <property name="text">
           <string>LOGAR</string>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>110</y>
            <width>141</width>
            <height>131</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>../_img/logo.jpg</pixmap>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="margin">
           <number>5</number>
          </property>
          <property name="indent">
           <number>0</number>
          </property>
          <property name="openExternalLinks">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring></cstring>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>500</y>
            <width>140</width>
            <height>40</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>NÃ£o tem cadastro?</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton">
          <property name="geometry">
           <rect>
            <x>270</x>
            <y>500</y>
            <width>71</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: rgb(85, 0, 255);
	border-radius: 3px;
}

QPushButton:hover {
	background-color: rgb(85, 0, 255);
	border: 3px solid rgb(148, 94, 255);
}
QPushButton:pressed{
	border: 2px solid rgb(255, 255, 255);
	color: rgb(0, 0, 0);
}</string>
          </property>
          <property name="text">
           <string>Cadastrar</string>
          </property>
         </widget>
         <widget class="QFrame" name="frame_5">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>359</y>
            <width>401</width>
            <height>61</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QPushButton" name="pushButton_3">
           <property name="geometry">
            <rect>
             <x>340</x>
             <y>20</y>
             <width>21</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>X</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>9</y>
             <width>280</width>
             <height>51</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
          <zorder>label_4</zorder>
          <zorder>pushButton_3</zorder>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>19</x>
            <y>20</y>
            <width>381</width>
            <height>91</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>../_img/letramento.png</pixmap>
          </property>
         </widget>
         <zorder>frame_5</zorder>
         <zorder>lineEdit</zorder>
         <zorder>lineEdit_2</zorder>
         <zorder>pushButton_2</zorder>
         <zorder>label</zorder>
         <zorder>label_3</zorder>
         <zorder>pushButton</zorder>
         <zorder>label_5</zorder>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_3">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>35</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Desenvolvido por RV::Technology Â©</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>837</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
